bin/run-lint

```diff
@@ -1,21 +1,42 @@
 #!/usr/bin/env node
-const child_process = require( 'child_process' );
+
 const path = require( 'path' );
-var args = [ '--cache', '--quiet', '--ext=.js',  '--ext=.jsx' ];
+const child_process = require( 'child_process' );
+const stylish = require( 'eslint/lib/formatters/stylish' );
+const exitCodeFromEslint = require( './eslint-formatters-calypso/src/lib/exit-code' );
+
+var args = [ '--cache', '--quiet', '--ext=.js', '--ext=.jsx' ];
+
+const argsBranchName = [ 'rev-parse', '--abbrev-ref', 'HEAD' ];
+const branchName = child_process.spawnSync( 'git', argsBranchName ).stdout.toString().trim();
+
+if ( branchName === 'master' ) {
```

Comment:
Why does it need to be a formatter? Can’t it be an `eslint` plugin, so that we can include it in the configuration?